rend3-egui/src/lib.rs

```diff
@@ -87,6 +88,72 @@ impl EguiRenderRoutine {
                 .unwrap();
         });
     }
+
+    /// Creates a wgpu texture
+    pub fn create_wgpu_texture(
+        renderer: &Arc<rend3::Renderer>,
+        format: wgpu::TextureFormat,
+        dimensions: (u32, u32),
+        lable: &str,
+    ) -> wgpu::Texture {
+        let device = &renderer.device;
```

Comment:
nit: let's just inline this variable so we just call `renderer.device.create_texture`